src/components/auth/auth.controller.js

```diff
@@ -0,0 +1,38 @@
+import authService from "./auth.service.js";
+
+const registerUser = async (req, res, next) => {
+  try {
+    const memberData = await authService.registerUser(req.body);
+    req.session.token = memberData.accessToken;
+    res.status(201).json(memberData);
+  } catch (error) {
+    next(error);
+  }
+};
+const loginUser = async (req, res, next) => {
+  try {
+    const tokenData = await authService.loginUser(req.body);
+    req.session.token = tokenData.accessToken;
```

Comment:
again same problem. either JWT base authentication aave, Or session based. te again banne no use karyo che.

akhu auth related logic change karvu pdse.

**mane akhu JWT related logic kadhi aap. and Session based banavi aap akhu.**

Reply:
session thi kyny thtu mne m htu k agl kyak use thse atle rkhyu 6 
ae kadhu nakhu to koi error ny ave mre